lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs

```diff
@@ -255,17 +255,17 @@
                     </div>
                 </dl>
           
-
-                <p>To access this sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
-
+                <% if (isMultipleSubServicesReq) { %>
+                    <p>To access your sub-services, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
```

Comment:
as per the prototype, `sign in to` should be `sign into` - please check with the content designer

Reply:
fixed with next checkin